org.nickvision.tagger.json

```diff
@@ -0,0 +1,89 @@
+{
+    "app-id" : "org.nickvision.tagger",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "org.nickvision.tagger",
+    "finish-args":[
+        "--socket=x11",
+        "--share=ipc",
+        "--filesystem=home",
```

Comment:
why access to complete `$HOME`?

Reply:
I guess I just need access to .config
So I would use `"--filesystem=home/.config"`?

Reply:
it should honour XDG_HOME_DATA, which unless you go out of your way to not use the GTK api, should just work.

Reply:
I use C++ std::filesystem apis.
So I can just remove the line completely?

Reply:
it's about where you harcode `~/.config` or use the standard glib / xdg API to get the directory.

https://docs.flatpak.org/en/latest/conventions.html#xdg-base-directories

Reply:
It's hardcoded. Therefore, I will include that line.

Reply:
You should edit your program to respect the XDG DIrectory specification

Reply:
```--persist=.config```